dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -118,6 +118,24 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationComboBox SetSelectedItem(String id);
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationComboBox2
```

Comment:
Can you follow up with our API reps on this? It will be a point of contention in the API review. Traditionally, we add more contracts and update the versioning in the existing runtime class. See wpnapps idl on how this is accomplished. 

Older properties that are not applicable anymore should go through a deprecation ritual. ie. You could say E_NOTIMPL  and throw an exception.

Reply:
Done. We're going to go the deprecation road. For now, the caller should see a warning that the old property is being deprecated. if the developer use the old property they will incur a copy as we have to convert data from the old structure to the new one and vice-versa. The good news is that we can't have more than 5 items, so the copy won't have any performance impact.